static/app/components/events/interfaces/frame/context.tsx

```diff
@@ -49,7 +51,10 @@ const Context = ({
   event,
   organization,
   className,
+  frameMeta,
+  registersMeta,
 }: Props) => {
+  console.log({frameMeta, registersMeta});
```

Comment:
```suggestion
```

Reply:
I removed it in my previous commit... sorry you got an alert before that 😬